### ISSUE TITLE
Black screen after toggling Full Screen on and off in Windows 10

### ISSUE THREAD

**Issue by Gromph** [godotengine/godot#48870]
**Godot version:**
3.3.1 32 and 64 bit
It also happens in: 3.2.3 64 bit, 3.3 64 bit

**OS/device including version:**
Windows 10 Pro 20H2 19042.985
AMD Ryzen 9 5900X
32GB Ram
Nvidia GTX 780   Driver version 466.47
Happens with both GLES2 and GLES3

So far I've only reproduced it on 1 computer, it did not happen on 4 others.

**Issue description:**
After toggling full screen mode on and off  the screen goes completely black

**Steps to reproduce:**
Run Godot and click New Project
Press Shift + F11 in the Editor to go to Full Screen
Press Shift + F11 again to exit
Result: The screen is now completely black

Pressing Alt + Tab, Windows Key, Alt + F4 does nothing, the screen is still black.

If you press Ctrl+Alt+Del it does show the windows full screen menu, then press cancel, everything draws normally.
From now on pressing Shift + F11 multiple times does work correctly.

**Minimal reproduction project:**
No Project needed, it's reproducible in the editor, and happens in games.

**Comment by Calinou** [comment]
@Gromph Do you have any third-party programs that could interfere with window management?

**Comment by Gromph** [comment]
Nope, I did more testing by creating a new account and made sure most third party processes were stopped.  It still happens.
